### ISSUE TITLE
Getting error when trying to visualize nvidia data: Error executing job with overrides:[]

### ISSUE THREAD

**Issue by lan-creator** [apple/ml-pgdvs#2]
Hello, when I try to run "Spatial Temporal Interpolation Visualizations" on the benchmark of nvidia dynamic scenes, I got a problem like this:

`
gnt:  91%|█████████ | 70/77 [2:55:22<17:32, 150.33s/it]
gnt:  92%|█████████▏| 71/77 [2:57:54<15:04, 150.74s/it]
gnt:  94%|█████████▎| 72/77 [3:00:22<12:30, 150.04s/it]
gnt:  95%|█████████▍| 73/77 [3:02:54<10:02, 150.56s/it]
gnt:  96%|█████████▌| 74/77 [3:05:23<07:30, 150.10s/it]
gnt:  97%|█████████▋| 75/77 [3:07:51<04:58, 149.32s/it]
gnt:  99%|█████████▊| 76/77 [3:10:19<02:29, 149.11s/it]
gnt: 100%|██████████| 77/77 [3:12:23<00:00, 149.92s/it]
Error executing job with overrides: ['verbose=true', 'distributed=false', 'seed=0', 'resume=vis_wo_resume', 'resume_dir=null', 'engine=visualizer_pgdvs', 'model=pgdvs_renderer', 'model.softsplat_metric_abs_alpha=100.0', 'static_renderer=gnt', 'static_renderer.model_cfg.ckpt_path=/data/code/ml-pgdvs/ckpts/gnt/model_720000.pth', 'series_eval=false', 'eval_batch_size=1', 'n_max_eval_data=-1', 'eval_save_individual=true', 'engine.engine_cfg.render_cfg.render_stride=1', 'engine.engine_cfg.render_cfg.chunk_size=2048', 'engine.engine_cfg.render_cfg.sample_inv_uniform=true', 'engine.engine_cfg.render_cfg.n_coarse_samples_per_ray=256', 'engine.engine_cfg.render_cfg.n_fine_samples_per_ray=0', 'engine.engine_cfg.render_cfg.mask_oob_n_proj_thres=1', 'engine.engine_cfg.render_cfg.mask_invalid_n_proj_thres=4', 'engine.engine_cfg.render_cfg.dyn_pcl_remove_outlier=true', 'engine.engine_cfg.render_cfg.dyn_pcl_outlier_knn=50', 'engine.engine_cfg.render_cfg.dyn_pcl_outlier_std_thres=0.1', 'engine.engine_cfg.render_cfg.gnt_use_dyn_mask=true', 'engine.engine_cfg.render_cfg.gnt_use_masked_spatial_src=false', 'engine.engine_cfg.render_cfg.dyn_render_use_flow_consistency=false', 'dataset=combined', 'dataset.dataset_list.train=[nvidia_eval]', 'dataset.dataset_list.eval=[nvidia_eval]', 'dataset.dataset_list.vis=[nvidia_vis]', 'dataset.dataset_specifics.mono_vis.scene_ids=[Balloon1]', 'dataset.data_root=/data/code/ml-pgdvs/data', 'n_dataloader_workers=1', 'dataset_max_hw=-1', 'dataset.use_aug=false', 'dataset.dataset_list.vis=[nvidia_vis]', 'dataset.dataset_specifics.nvidia_vis.scene_ids=[Balloon1]', 'vis_specifics.n_render_frames=400', 'vis_specifics.vis_center_time=50', 'vis_specifics.vis_time_interval=50', 'vis_specifics.vis_bt_max_disp=32']
Traceback (most recent call last):
  File "/data/code/ml-pgdvs/pgdvs/run.py", line 267, in <module>
    cli()
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/main.py", line 90, in decorated_main
    _run_hydra(
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/_internal/utils.py", line 389, in _run_hydra
    _run_app(
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/_internal/utils.py", line 452, in _run_app
    run_and_report(
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/_internal/utils.py", line 216, in run_and_report
    raise ex
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/_internal/utils.py", line 213, in run_and_report
    return func()
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/_internal/utils.py", line 453, in <lambda>
    lambda: hydra.run(
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/_internal/hydra.py", line 132, in run
    _ = ret.return_value
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/core/utils.py", line 260, in return_value
    raise self._return_value
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/hydra/core/utils.py", line 186, in run_job
    ret.return_value = task_function(task_cfg)
  File "/data/code/ml-pgdvs/pgdvs/run.py", line 263, in cli
    run(cfg, hydra_config)
  File "/data/code/ml-pgdvs/pgdvs/run.py", line 192, in run
    return _distributed_worker(
  File "/data/code/ml-pgdvs/pgdvs/run.py", line 146, in _distributed_worker
    output = engine.run()
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/data/code/ml-pgdvs/pgdvs/engines/visualizer_pgdvs.py", line 26, in run
    self.vis_model()
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/data/code/ml-pgdvs/pgdvs/engines/visualizer_pgdvs.py", line 91, in vis_model
    ret_dict = self._get_model_module(self.model).forward(
  File "/data/code/ml-pgdvs/pgdvs/renderers/pgdvs_renderer.py", line 146, in forward
    (render_dyn_rgb, render_dyn_mask, render_dyn_info) = self.dyn_renderer(
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/data/code/ml-pgdvs/pgdvs/renderers/pgdvs_renderer_dyn.py", line 184, in forward
    splat_dyn_img_full, softsplat_metric_src1_to_src2 = self.softsplat_img(
  File "/data/code/ml-pgdvs/pgdvs/renderers/pgdvs_renderer_base.py", line 80, in softsplat_img
    splat_img_src1_to_tgt = softsplat.softsplat(
  File "/data/code/ml-pgdvs/pgdvs/utils/softsplat.py", line 311, in softsplat
    tenOut = softsplat_func.apply(tenIn, tenFlow)
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/torch/autograd/function.py", line 539, in apply
    return super().apply(*args, **kwargs)  # type: ignore[misc]
  File "/data/apps/anaconda3/envs/pgdvs/lib/python3.10/site-packages/torch/cuda/amp/autocast_mode.py", line 121, in decorate_fwd
    return fwd(*args, **kwargs)
  File "/data/code/ml-pgdvs/pgdvs/utils/softsplat.py", line 421, in forward
    assert False
AssertionError
`

I have followed the instruction of "https://github.com/apple/ml-pgdvs/blob/main/docs/BENCHMARK_NVIDIA.md" , and I just want to show the visuable result. I guess this may be related to the config process, but how could I solve this problem? Thank you!

Environment: CUDA 11.8 + nvidia_A800 and without jax

**Comment by Xiaoming-Zhao** [comment]
Hi, based on the traceback, it throws error at https://github.com/apple/ml-pgdvs/blob/13f9875eb0b7de6068452ec9c37231eda48581df/pgdvs/utils/softsplat.py#L420-L421

It means that the data is not on GPU.

It seems like your code is not running with a GPU, which is also reflected on your progress bar for running GNT: if I read it correctly, it takes ~3 hour to render an image.

Do you mind double checking?

**Comment by lan-creator** [comment]
Wow cool! I first thoght it was the problem of hydra as I'm not familiar with it. And I checked the environment, where I found that in the envs/pgdvs.yaml conda always installs cpu-version torch so I couldn't use cuda. I guess this is conda channels problem, and I finally change dependencies like this and solve the problem:
`dependencies:
  - python=3.10
  - pytorch=2.0.1=py3.10_cuda11.8_cudnn8.7.0_0
  - torchaudio=2.0.2=py310_cu118
  - torchvision=0.15.2=py310_cu118
  - pytorch-cuda=11.8`
